cypress/support/index.d.ts

```diff
@@ -1,9 +1 @@
 import '@mate-academy/cypress-tools/support';
```

Comment:
The import statement is not needed if you are not using any functionality from '@mate-academy/cypress-tools/support'. If you are using it, make sure you have installed this package in your project.